atomicops.h

```diff
@@ -662,10 +662,10 @@ namespace moodycamel
 		        }
 		    }
 		    
-		    ssize_t availableApprox() const AE_NO_TSAN
+		    size_t availableApprox() const AE_NO_TSAN
```

Comment:
Looks good, but should be `std::size_t` here and below just in case.

Reply:
Right. You are already using `std::size_t` in other parts.